starknet-foundry/crates/cast/src/helpers/scarb_utils.rs

```diff
@@ -31,30 +31,34 @@ fn get_property(
             )),
         None => profiled
             .and_then(|t| t.get(property))
-            .and_then(serde_json::Value::as_str)
+            .and_then(Value::as_str)
             .map(String::from)
             .ok_or(anyhow!("Property not found in tool: {property}")),
     }
 }
 
 pub fn parse_scarb_config(
     profile: &Option<String>,
-    path: Option<Utf8PathBuf>,
+    path: &Option<Utf8PathBuf>,
 ) -> Result<CastConfig> {
-    let manifest_path = find_manifest_path(path.as_ref().map(camino::Utf8PathBuf::as_path))
-        .expect("Failed to obtain Scarb.toml file path");
-
-    if let Some(manifest_path) = path {
-        if !manifest_path.exists() {
-            bail! {"{manifest_path} file does not exist!"};
-        };
+    let found_path = find_manifest_path(path.as_ref().map(Utf8PathBuf::as_path));
```

Comment:
are you sure this works when `path` is `None`?

Reply:
It should, Scarb searches for the manifest file up the working tree